test/list.js

```diff
@@ -0,0 +1,56 @@
+import { getDocumentLocaleSettings } from '../lib/common.js';
+import { getSeparator } from '../lib/list.js';
+
+const expect = chai.expect;
+
+describe('getSeparator', () => {
+
+	const documentLocaleSettings = getDocumentLocaleSettings();
+
+	afterEach(() => documentLocaleSettings.reset());
+
+	[
+		{ locale: 'ar', expect: [' ، ', '\xa0،\xa0'] },
+		{ locale: 'ar-SA', expect: [' ، ', '\xa0،\xa0'] },
+		{ locale: 'cy-GB', expect: [', ', ',\xa0'] },
+		{ locale: 'da', expect: [', ', ',\xa0'] },
+		{ locale: 'da-DK', expect: [', ', ',\xa0'] },
+		{ locale: 'de', expect: [', ', ',\xa0'] },
+		{ locale: 'de-DE', expect: [', ', ',\xa0'] },
+		{ locale: 'en', expect: [', ', ',\xa0'] },
+		{ locale: 'en-CA', expect: [', ', ',\xa0'] },
+		{ locale: 'en-GB', expect: [', ', ',\xa0'] },
+		{ locale: 'en-US', expect: [', ', ',\xa0'] },
+		{ locale: 'es', expect: [', ', ',\xa0'] },
+		{ locale: 'es-ES', expect: [', ', ',\xa0'] },
+		{ locale: 'es-MX', expect: [', ', ',\xa0'] },
+		{ locale: 'fr', expect: [', ', ',\xa0'] },
+		{ locale: 'fr-CA', expect: [', ', ',\xa0'] },
+		{ locale: 'fr-FR', expect: [', ', ',\xa0'] },
+		{ locale: 'fr-ON', expect: [', ', ',\xa0'] },
+		{ locale: 'hi', expect: [', ', ',\xa0'] },
+		{ locale: 'hi-IN', expect: [', ', ',\xa0'] },
+		{ locale: 'ja', expect: ['、', '、'] },
+		{ locale: 'ja-JP', expect: ['、', '、'] },
+		{ locale: 'ko', expect: [', ', ',\xa0'] },
+		{ locale: 'ko-KR', expect: [', ', ',\xa0'] },
+		{ locale: 'nl', expect: [', ', ',\xa0'] },
+		{ locale: 'nl-NL', expect: [', ', ',\xa0'] },
+		{ locale: 'pt', expect: [', ', ',\xa0'] },
+		{ locale: 'pt-BR', expect: [', ', ',\xa0'] },
+		{ locale: 'sv', expect: [', ', ',\xa0'] },
+		{ locale: 'sv-SE', expect: [', ', ',\xa0'] },
+		{ locale: 'tr', expect: [', ', ',\xa0'] },
+		{ locale: 'tr-TR', expect: [', ', ',\xa0'] },
+		{ locale: 'zh', expect: ['、', '、'] },
+		{ locale: 'zh-CN', expect: ['、', '、'] },
+		{ locale: 'zh-TW', expect: ['、', '、'] },
```

Comment:
This is more of an aside (non-blocking side question), but how come we support [35 locales/variants here](https://github.com/BrightspaceUI/intl/blob/main/test/fileSize.js#L36-L70) but we only have [only 18 lang files](https://github.com/BrightspaceUI/core/tree/main/lang) in our core repo?

Reply:
FWIW, these tests helped me catch three cases I hadn't accounted for:

- `zh` (had only checked specifically `zh-cn` and `zh-tw`)
- `ar-sa`
- `ja-jp`

Reply:
Yeah sometimes the base locale tests are handy because technically our clients can create whatever locale they want in the system and call it `en-zz` (or `fr-on`...) and this library should fall back gracefully.

Reply:
Hmm, so currently I've just hard-coded the special-case locales, including the base and sub-locales.

Should I instead check the base locale directly? I see that `fileSize` does:

```js
	const subtags = language.split('-');
	const baseLanguage = subtags[0];
```